src/html/HtmlReporter.js

```diff
@@ -517,7 +517,7 @@ jasmineRequire.HtmlReporter = function(j$) {
             specDescription = 'SPEC HAS NO EXPECTATIONS ' + specDescription;
           }
           if (resultNode.result.status === 'pending') {
-            if (resultNode.result.pendingReason !== '') {
+            if (resultNode.result.pendingReason) {
```

Comment:
I'm pretty sure this is unnecessary. If it is necessary, it might be a symptom of an undocumented breaking change to the reporter API. In any case, there doesn't appear to be any test coverage for it.